ansible_ai_connect/main/settings/base.py

```diff
@@ -679,14 +689,17 @@ def is_ssl_enabled(value: str) -> bool:
         },
     }
 
+
+
+
+
 # Lazy import to avoid circular dependencies
 from ansible_ai_connect.ai.api.model_pipelines.pipelines import MetaData  # noqa
 from ansible_ai_connect.ai.api.model_pipelines.registry import REGISTRY_ENTRY  # noqa
 
-pipelines = list(filter(lambda p: issubclass(p, MetaData), REGISTRY_ENTRY.keys()))
-pipeline_config: dict = {}
-for pipeline in pipelines:
-    pipeline_config[pipeline.__name__] = ANSIBLE_AI_WCA_CONFIG
+pipelines = [i for i in REGISTRY_ENTRY.keys() if issubclass(i, MetaData)]
+pipeline_config: dict = {k.__name__: ANSIBLE_AI_WCA_CONFIG for k in pipelines}
+print(f"pipeline_confi={pipeline_config}")
```

Comment:
I'm OK with this.. but you've made no friends with Copilot.

But you could spell `pipeline_config` correctly (you have `pipeline_confi`).

Reply:
oh, let's me clean my mess :-).